custom/icds_reports/reports/disha.py

```diff
@@ -73,7 +75,7 @@ def _get_columns(self):
 
     def _get_rows(self, query_master=False):
         if query_master:
-            objects = DishaIndicatorView.objects.using('icds-ucr')
+            objects = DishaIndicatorView.objects.using(get_icds_ucr_citus_db_alias())
         else:
             objects = DishaIndicatorView.objects
```

Comment:
I assume this is not changed, since Citus is the default, is it?

Reply:
This will be routed based on the django DB router which should be routing to Citus by default now.